src/elements/content-explorer/stories/tests/PreviewDialog-visual.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { useArgs } from '@storybook/preview-api';
+import { Button } from '@box/blueprint-web';
+import { userEvent, within } from '@storybook/test';
+
+import { addRootElement } from '../../../../utils/storybook';
+
+import PreviewDialog from '../../PreviewDialog';
+
+// need to import this into the story because it's usually in ContentExplorer
+import '../../../common/modal.scss';
+
+export const basic = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        const button = canvas.getByRole('button', { name: 'Launch PreviewDialog' });
+        await userEvent.click(button);
+        expect(canvas.getByText('Book Sample.pdf')).toBeInTheDocument();
+    },
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    render: (args: any) => {
```

Comment:
we should do something about this too. no point in having the rule if we're just disabling it